### PR TITLE
ENH: minor filter algorithm consistency updates

### PR DESCRIPTION
**Pull request by ambrosejcarr**
* gaussian_high_pass
* gaussian_low_pass
* mean_high_pass
* richardson_lucy_deconvolution

Minor updates to: 
- line length
- documentation
- adding types
- moving verbose to `run`

### CODE REVIEW

File: starfish/pipeline/filter/gaussian_high_pass.py [hunk 1]
```diff
@@ -9,29 +8,30 @@
...
+from starfish.typing import Number
 from ._base import FilterAlgorithmBase
 
 
 class GaussianHighPass(FilterAlgorithmBase):
 
     def __init__(
-            self, sigma: Union[Number, Tuple[Number]], is_volume: bool=False, verbose: bool=False, **kwargs
-    ) -> None:
+            self, sigma: Union[Number, Tuple[Number]], is_volume: bool=False, verbose: bool=False,
```
Comment (ttung) [inline starfish/pipeline/filter/gaussian_high_pass.py]: style, like the other ones.  avoid backslash if possible.

File: starfish/pipeline/filter/gaussian_high_pass.py [hunk 2]
```diff
@@ -67,7 +68,8 @@ def high_pass(image: np.ndarray, sigma: Union[Number, Tuple[Number]]) -> np.ndar
 
         """
         if image.dtype != np.uint16:
-            DataFormatWarning('gaussian filters currently only support uint16 images. Image data will be converted.')
+            DataFormatWarning('gaussian filters currently only support uint16 images. Image data '
```
Comment (ttung) [inline starfish/pipeline/filter/gaussian_high_pass.py]: double quotes!

File: starfish/pipeline/filter/gaussian_high_pass.py [hunk 3]
```diff
@@ -78,7 +80,9 @@ def high_pass(image: np.ndarray, sigma: Union[Number, Tuple[Number]]) -> np.ndar
 
         return filtered
 
-    def run(self, stack: ImageStack, in_place: bool=True) -> Optional[ImageStack]:
+    def run(
```
Comment (ttung) [inline starfish/pipeline/filter/gaussian_high_pass.py]: style

File: starfish/pipeline/filter/gaussian_low_pass.py [hunk 4]
```diff
@@ -70,12 +70,14 @@ def low_pass(image: np.ndarray, sigma: Union[Number, Tuple[Number]]) -> np.ndarr
 
         """
         if image.dtype != np.uint16:
-            DataFormatWarning('gaussian filters only support uint16 images. Image data will be converted')
+            DataFormatWarning(
```
Comment (ttung) [inline starfish/pipeline/filter/gaussian_low_pass.py]: double quotes!

File: starfish/pipeline/filter/gaussian_low_pass.py [hunk 5]
```diff
@@ -70,12 +70,14 @@ def low_pass(image: np.ndarray, sigma: Union[Number, Tuple[Number]]) -> np.ndarr
...
+                'gaussian filters only support uint16 images. Image data will be converted')
             image = img_as_uint(image)
 
         blurred = gaussian(
-            image, sigma=sigma, output=None, cval=0, multichannel=True, preserve_range=True, truncate=4.0)
-
+            image, sigma=sigma, output=None, cval=0, multichannel=True, preserve_range=True,
```
Comment (ttung) [inline starfish/pipeline/filter/gaussian_low_pass.py]: image on first line, rest on second.

File: starfish/pipeline/filter/mean_high_pass.py [hunk 6]
```diff
@@ -77,7 +74,8 @@ def high_pass(image: np.ndarray, size: Number) -> np.ndarray:
 
         """
         if image.dtype != np.uint16:
-            DataFormatWarning('mean filters currently only support uint16 images. Image data will be converted.')
+            DataFormatWarning(
+                'Mean filters currently only support uint16 images. Image data will be converted.')
```
Comment (ttung) [inline starfish/pipeline/filter/mean_high_pass.py]: quotes!

File: starfish/pipeline/filter/mean_high_pass.py [hunk 7]
```diff
@@ -88,7 +86,9 @@ def high_pass(image: np.ndarray, size: Number) -> np.ndarray:
 
         return filtered
 
-    def run(self, stack: ImageStack, in_place: bool=True) -> Optional[ImageStack]:
+    def run(
+            self, stack: ImageStack, in_place: bool=True, verbose: bool=False) \
```
Comment (ttung) [inline starfish/pipeline/filter/mean_high_pass.py]: style

File: starfish/pipeline/filter/richardson_lucy_deconvolution.py [hunk 8]
```diff
@@ -8,60 +8,65 @@
...
+            compatibility
 
         """
         self.num_iter = num_iter
         self.sigma = sigma
         self.clip = clip
         self.kernel_size: int = int(2 * np.ceil(2 * sigma) + 1)
-        self.psf: np.ndarray = gaussian_kernel(shape=(self.kernel_size, self.kernel_size), sigma=sigma)
-        self.verbose = verbose
+        self.psf: np.ndarray = gaussian_kernel(shape=(self.kernel_size, self.kernel_size),
```
Comment (ttung) [inline starfish/pipeline/filter/richardson_lucy_deconvolution.py]: prefer the no args on the first line if you wrap, i.e.,

```
self.psf: np.ndarray = gaussian_kernel(
    shape=(self.kernel_size, self.kernel_size), sigma=sigma)
```

Comment (ambrosejcarr) [reply starfish/pipeline/filter/richardson_lucy_deconvolution.py]: 👍 I split it out over a few lines, one arg per line.

**Pull request merged**